examples/lock-app/nrfconnect/main/AppTask.cpp

```diff
@@ -418,16 +393,11 @@ void AppTask::StartBLEAdvertisementHandler(AppEvent * aEvent)
 }
 
 #ifdef CONFIG_CHIP_NFC_COMMISSIONING
-void AppTask::ThreadProvisioningHandler(const ChipDeviceEvent * event, intptr_t arg)
+void AppTask::ThreadProvisioningHandler(const ChipDeviceEvent * event, intptr_t)
 {
-    ARG_UNUSED(arg);
-    if ((event->Type == DeviceEventType::kServiceProvisioningChange) && ConnectivityMgr().IsThreadProvisioned())
+    if (event->Type == DeviceEventType::kCHIPoBLEAdvertisingChange && event->CHIPoBLEAdvertisingChange.Result == kActivity_Stopped)
     {
-        const int result = sNFC.StopTagEmulation();
-        if (result)
-        {
-            LOG_ERR("Stopping NFC Tag emulation failed");
-        }
+        NFCMgr().StopTagEmulation();
```

Comment:
Please add checking if tag is started before trying to stop. There is a bug in the code, that when you use test mode (button 3) without previously pressing button 4, we try to stop tag that is not started and it results in error printed in logs. I pushed PR fixing it: https://github.com/project-chip/connectedhomeip/pull/5655

Reply:
I added the check inside `StopTagEmulation()`.

Reply:
Oh right, I missed it